core/modules/system/system.install

```diff
@@ -203,12 +203,40 @@ function system_requirements($phase) {
     $tasks = new $class();
     $dbinfo = $t('!system version !version', array('!system' => $tasks->name(), '!version' => Database::getConnection()->version()));
     $requirements['database_system'] = array(
-      'title' => $t('MySQL Database'),
+      'title' => $t('MySQL database'),
       'severity' => REQUIREMENT_INFO,
       'value' => $dbinfo,
     );
   }
 
+  // Report cURL version or absence.
+  if (function_exists('curl_init')) {
+    $curl_version = curl_version();
+    if (is_array($curl_version)) {
+      $version_info = array();
+      if (!empty($curl_version['version'])) {
+        $version_info[] = $t('Version:') . ' ' . $curl_version['version'];
+      }
+      if (!empty($curl_version['ssl_version'])) {
+        $version_info[] = $t('SSL version:') . ' ' . $curl_version['ssl_version'];
+      }
+      if (!empty($curl_version['libssh_version'])) {
+        $version_info[] = $t('libssh version:') . ' ' . $curl_version['libssh_version'];
+      }
+      $curl_status = implode(', ', $version_info);
+    }
+    else {
+      $curl_status = $t('cURL version not available');
```

Comment:
This makes it sound as if this is a problem. I would simply leave that as `$t('Enabled')` or `$t('Available')`. The version is a nice to have information - not necessary in any way, and not a problem if we cannot determine it.

Reply:
I was unable to find in any documentation the circumstances under which PHP's `curl_version()` would exist, but would return `FALSE` instead of an array. If it returns FALSE, can we be assured that cURL exists?

Reply:
It looks like it returns FALSE if curl is not installed, so that's a better check for its existence. Reordered the logic to reflect that.